lib/atomic_web/plugs/authorize.ex

```diff
@@ -45,7 +45,7 @@ defmodule AtomicWeb.Plugs.Authorize do
     role = Organizations.get_role(user.id, organization_id)
     allowed_roles = Organizations.roles_bigger_than_or_equal(minimum_authorized_role)
 
-    (organization_id in user_organizations && role in allowed_roles) || user.role == :admin
+    organization_id in user_organizations && role in allowed_roles
```

Comment:
This is needed here, in order to allow users of type `:master` to bypass this authorization check.
```suggestion
    (organization_id in user_organizations && role in allowed_roles) || user.role == :master
```